modules/apps/object/object-service/service.xml

```diff
@@ -526,6 +526,7 @@
 
 		<column name="objectViewId" type="long" />
 		<column name="objectFieldName" type="String" />
+		<column localized="true" name="label" type="String" />
```

Comment:
@p-albuquerque can you sort this? Right after `objectFieldName`

Reason: First, we have audit fields... then ids... then custom fields.